docs/communications/jobs-tasks/jobs-vs-tasks.md

```diff
@@ -0,0 +1,23 @@
+# Jobs vs tasks
+
+A vehicle's *job* (or profession) describes the type of *tasks* it can perform.
+
+For example, an important part of the project is the ISR Search mission. To perform this mission,
+the GCS look for vehicles that have an `isrSearch` job. By having the `isrSearch` job, the vehicle
+is stating that it is capable of performing all of the following tasks that are related to the job:
+
+  - `takeoff` task
+  - `loiter` task
+  - `isrSearch` task
+  - `land` task
+
+The GCS will assign the vehicles the `isrSearch` job by sending them a [start message][]. By doing
```

Comment:
I'm assuming [start message] was supposed to link to the start message format page?

Reply:
Fixed